great-idea/index.html

```diff
@@ -1,3 +1,132 @@
+
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+
+  <title>Great Idea!</title>
+
+  <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
+  <link rel="stylesheet" href="css/index.css">
+
+  <!--[if lt IE 9]>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
+  <![endif]-->
+</head>
+
+
+
+
+
+
+<!--                   dont touch above                  -->
+
+
+
+<body>
+
+<container>
+
+  <nav> 
+    <div>
+      <a class="navbar" href="services.html">Services</a>
+      <a class="navbar" href="#">Product</a>
+      <a class="navbar" href="#">Vision</a>
+      <a class="navbar" href="#">Features</a>
+      <a class="navbar" href="#">About</a>
+      <a class="navbar" href="#">Contact</a>
+     <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+    </div>
+  </nav>
+  
+  
+<header>
+  
+  <div class="headerLeft">
+    Innovation<br>
+    On<br>
+    Demand <br>  
+    <button>
+      Get Started
```

Comment:
Try getting the style similar to how the design file is.

Look up how to remove button styles in css.

You can also make it into plain text with a 1px solid black border.